### PR TITLE
Updated dog profile swipe features

### PR DESCRIPTION
**Pull request by Brenda3116**

### CODE REVIEW

File: src/front/js/component/DogList.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            'Authorization': `Bearer ${localStorage.getItem('token')}`
+          }
+        });
+        const data = await response.json();
+        setDogs(data.profiles);
+        setLoading(false);
+      } catch (error) {
+        console.error('Error fetching dog profiles:', error);
+      }
+    };
```
Comment (CHCHALLE) [inline src/front/js/component/DogList.js:22]: use fetchUserProfile from flux file rather than fetchDogProfiles

**Review by CHCHALLE — CHANGES_REQUESTED** [review]
delete 9-22 add fetchUserProfile

**Review by CHCHALLE — APPROVED** [review]
looks good

File: src/front/js/component/DogList.js (hunk 1 above)
Comment (Brenda3116) [reply src/front/js/component/DogList.js:22]: DogList.js now simply calls the actions from Flux

**Pull request merged**